### DISCUSSION TITLE
Micro Frontend architect

### DISCUSSION THREAD

**Discussion by maczyt** [maczyt/loonflow#3] [Ideas]
使用微前端架构来管理维护，便于支持多种技术栈

**Comment by maczyt** [comment]
[what is Micro Frontend](https://swearer23.github.io/micro-frontends/)

**Comment by maczyt** [comment]
or 模块联邦？

**Comment by blackholll** [comment]
现阶段不建议使用微前端架构， 会增加维护成本，后端也才一个服务。 开发人员也就一个前端一个后端。弊大于利
